File: app/src/main/java/com/gavin/asmdemo/db/room/AppDatabase.java
package com.gavin.asmdemo.db.room;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.gavin.asmdemo.db.room.entity.User;
import com.gavin.asmdemo.db.room.entity.UserDao;

@Database(entities = {User.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();
}

File: jetpackpokemon/src/main/java/com/github/pokemon/local/AppDataBase.kt
package com.github.pokemon.local

import androidx.room.Database
import androidx.room.RoomDatabase
import com.github.pokemon.data.entity.PokeEntity

@Database(
        entities = [PokeEntity::class],
        version = 1, exportSchema = false
)
abstract class AppDataBase : RoomDatabase() {
    abstract fun pokemonDao(): PokemonDao
}